### DISCUSSION TITLE
Which Go Version work?

### DISCUSSION THREAD

**Discussion by itgeniusshuai** [sijms/go-ora#149] [Show and tell]
1. I use go version 1.15, when I `go get `,show `..\github.com\sijms\go-ora\v2\wallet.go:51:19: undefined: os.ReadFile`.The version 1.15 don't have this function! 
2. Is it a client that doesn't requied install Instant Client Package?

**Comment by sijms** [comment]
1- I see your pull request that fix Golang version dependencies to 1.15
2- the package doesn't need instant client, another package dependencies or environmental variables like NLS_LANG.
just download the package and see examples. You will be able to connect and work with oracle

**Reply by hmmftg** [reply]
it's the best oracle driver I've ever seen
better than drivers which oracle wrote itself
and the only real cross-platform driver for oracle
when we choose golang, dependencies like oracle client sh*t is really painful

**Reply by itgeniusshuai** [reply]
I agree with you,I search one day for golang oracle driver one day on github, I choose this.it's the best one！！！
